cms/src/components/content/ContentOverview.vue

```diff
@@ -179,19 +184,42 @@ const handleFilterDelete = async (event: Event) => {
     console.log(filters.value);
 };
 
+const handleDeleteAllFilters = () => {
+    filters.value = [];
+    queryOptions.value.publishStatus = "all";
+    queryOptions.value.translationStatus = "all";
+    queryOptions.value.tagType = tagType;
+};
+
 watch(sortOption, () => {
     queryOptions.value.orderBy = sortOption.value;
 });
 
-watch(filters, () => {
+watch(filters.value, () => {
+    console.log("filters changed");
     filters.value.map((filter) => {
-        if (filter.filter == "translation")
+        if (filter.filter == "translation") {
+            console.log(`Filter ${filter.filter} by ${filter.by}`);
             queryOptions.value.translationStatus = filter.by as
                 | "all"
                 | "translated"
                 | "untranslated"
                 | undefined;
-        console.log(filter);
+        } else if (filter.filter == "type") {
+            console.log(`Filter ${filter.filter} by ${filter.by}`);
+            queryOptions.value.tagType = filter.by as TagType;
+        } else if (filter.filter == "tag") {
+            queryOptions.value.tags?.push(filter.by as string);
+        } else if (filter.filter == "status") {
+            console.log(`Filter ${filter.filter} by ${filter.by}`);
+            queryOptions.value.publishStatus = filter.by as
+                | "all"
+                | "published"
+                | "scheduled"
+                | "expired"
+                | "draft"
+                | undefined;
+        }
     });
 });
```

Comment:
@ivanslabbert  @laurensgroeneveld  This is how the filtering is currently working. It works to a degree but I feel there can be a better solution than doing it this way. As this method is buggy.